### PR TITLE
Try Python 3/6

### PR DESCRIPTION
**Pull request by ocefpaf**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -41,7 +41,7 @@ build:
   features:
     - vc9  # [win and py27]
     - vc10  # [win and py34]
-    - vc14  # [win and py35]
+    - vc14  # [win and py>35]
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Is `vc14` the correct option for Python 3.6?

**Comment by ccordoba12** [comment]
Please skip `osx` to avoid wasting valuable build time on it. I mean, we know it'll always time out :-)

File: recipe/meta.yaml (hunk 1 above)
Comment (ccordoba12) [inline recipe/meta.yaml]: I believe so, yes :-)

**Comment by ocefpaf** [comment]
> Please skip osx to avoid wasting valuable build time on it. I mean, we know it'll always time out :-)

Agreed. I will leave that there for now though because people are working on it and I do not want them to deal with conflicts.

**Pull request merged**